sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/OSPatchingStatus.cs

```diff
@@ -0,0 +1,82 @@
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="OSPatchingStatus"/>. </summary>
+        internal OSPatchingStatus()
+        {
+            OSPatchingErrors = new ChangeTrackingList<MachineLearningError>();
+        }
+
+        /// <summary> Initializes a new instance of <see cref="OSPatchingStatus"/>. </summary>
+        /// <param name="patchStatus"> The os patching status. </param>
+        /// <param name="latestPatchTime"> Time of the latest os patching. </param>
+        /// <param name="rebootPending"> Specifies whether this compute instance is pending for reboot to finish os patching. </param>
+        /// <param name="scheduledRebootTime"> Time of scheduled reboot. </param>
+        /// <param name="osPatchingErrors"> Collection of errors encountered when doing os patching. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal OSPatchingStatus(PatchStatus? patchStatus, string latestPatchTime, bool? rebootPending, string scheduledRebootTime, IReadOnlyList<MachineLearningError> osPatchingErrors, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            PatchStatus = patchStatus;
+            LatestPatchTime = latestPatchTime;
+            RebootPending = rebootPending;
+            ScheduledRebootTime = scheduledRebootTime;
+            OSPatchingErrors = osPatchingErrors;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The os patching status. </summary>
+        public PatchStatus? PatchStatus { get; }
+        /// <summary> Time of the latest os patching. </summary>
+        public string LatestPatchTime { get; }
```

Comment:
Could this be a type of `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).

Reply:
resolved